codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/util/ConfigProperty.kt

```diff
@@ -212,6 +213,15 @@ class ConfigProperty private constructor(builder: Builder) {
             }
         }
 
+        /**
+         * Shorthand to declare that the property has a [symbol] where the builder version is the same, but nullable.
+         */
+        fun useSymbolWithNullableBuilder(symbol: Symbol) {
+            this.symbol = symbol
+            this.builderSymbol = symbol.asNullable()
+            this.toBuilderExpression = "" // nothing needed to convert back, T fits into T?
+        }
```

Comment:
**Question**: It looks like every place this is used, one must also supply a `propertyType = ConfigPropertyType.RequiredWithDefault("false"|"true")`. Should the default setting just be added to this method?